README.md

```diff
@@ -189,6 +189,21 @@ You will also see the advantages of using R4ML, which is a git-downloadable open
 
 ## Sample output
 
```

Comment:
I believe we need to have the score screen shots. Since it is main point of this cp. Something along the line of 

"
cores
We can also see the predicted scores. The higher the score, the higher the margin and the better or more confident we are of the prediction. (see cell 19)

In [19]:
head(svm_preds_1$scores)
class:1    class:2
1.8097469    -1.8097469
1.6077980    -1.6077980
1.6757843    -1.6757843
-0.1936054    0.1936054
1.1191495    -1.1191495
1.5759885    -1.5759885

Some explaination

for example 
row 1 has better margin than row 2 and hence sample at row 1 prediction's confidence level is high

note that row4 , numbers are low and hence confidence is low and infact that row was misclassified (since class1 has -ve sign and class2 has positive sign )
"

Reply:
I didn't notice that pic for scores are there. Probably some explaination as in previous comments would be nice

Reply:
@aloknsingh - made some changes. Please take a look.

Reply:
LGTM . Thanks a lot @rhagarty and @stevemart .